### DISCUSSION TITLE
Sea-spray velocity-dependent or prescribed ?

### DISCUSSION THREAD

**Discussion by jchylik** [ARM-Development/comble-mip#67] [Q&A] [answered]
Although adding sea-surface emissions has been relatively easy, its is unclear whether we should use a) prescribed values OR b) calculate interactively. The page [Specifying Surface Aerosol Emissions for Simulations with Multi-Modal Aerosol](https://arm-development.github.io/comble-mip/notebooks/setup/aerosol-surface.html#jaegle-et-al-2011-surface-particle-flux-integrated-over-all-sizes) refers to formula from _Jaegle et al. (2011)_, but is ambiguous about the the impact values 

a) prescribed values at the webpage (in the code example)
```
SST = 5
vh = 20
```

b) interactive (as per the last paragraph)
```
SST =  ts [:]  
vh ~~  estimated by logarithmic profile between the surface and the velocity at 20 m
```

**Comment by floriantornow** [comment, marked as answer]
Please calculate interactive sea-surface emissions (i.e., as a function of prevalent SST and 10m windspeed). Apologies for the lack of clarity here and please find the page updated accordingly.

Thank you,
Florian

**Reply by jchylik** [reply]
Thank you for the response.

>   function of prevalent SST and 10m windspeed

 
 That still raises one more question:
 
How to estimate 10 m windspeed? The first grid level is 20 m (or 15 m). Shall we assume a logarithmic or linear wind profile in between?

**Comment by floriantornow** [comment]
The altitudes of the layer faces are defined by zw_grid, with the first one above the surface at 20m (i.e., grid cell center at 10m). So there should be u, v wind components at the faces of the grid cell center (same as WRF) and no need to interpolate. Can you please check your setup and let me know?

**Reply by jchylik** [reply]
> layer faces are defined by zw_grid, with the first one above the surface at 20m (i.e., grid cell center at 10m)

Oh, I see, I have misunderstood that the values in  zw_grid were lower cell faces.
